src/gui/accountstate.cpp

```diff
@@ -336,30 +297,25 @@ void AccountState::tagLastSuccessfullETagRequest(const QDateTime &tp)
 
 void AccountState::checkConnectivity(bool blockJobs)
 {
+    //>>>>> These are all pre-checks that may effectively cancel the check
```

Comment:
we should not use `<<<<<` or `>>>>>>` ... this is used for declaring conflicts in git - this triggers me ;-)